modules/dragster/src/main/ts/ephox/dragster/datatransfer/DragImage.ts

```diff
@@ -0,0 +1,24 @@
+import { Id, Optional } from '@ephox/katamari';
+
+interface DragImageData {
+  image: Element;
+  x: number;
+  y: number;
+}
```

Comment:
Maybe indicate that this is to be an immutable structure.

```suggestion
interface DragImageData {
  readonly image: Element;
  readonly x: number;
  readonly y: number;
}
```